orttraining/orttraining/test/session/training_api_tests.cc

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#if defined(ENABLE_TRAINING) && defined(ENABLE_TRAINING_ON_DEVICE)
```

Comment:
Could you plese put this file into https://github.com/microsoft/onnxruntime/tree/training_dev/on_device_poc/orttraining/orttraining/test/training_api/module

Reply:
moving